### DISCUSSION TITLE
/intro.html

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#3] [Announcements]
# /intro.html

Van Blog 的官方网站

https://vanblog.mereith.com/intro.html

**Comment by redgreat** [comment]
TODO里的数据库能加个ORACLE，只是非结构化的话应该还好。

**Reply by Mereithhh** [reply]
查了一下，打算加上的 TypeORM 默认支持 Oracle 的

**Comment by mmm141023** [comment]
使用作者的van-blog搭建的前端面试题汇总的网站，性能很好，界面清新，大家可以体验下https://www.whyknown.com/

**Comment by coderyjw** [comment]
一件部署，非常好用。
https://yejiwei.com/

**Reply by coderyjw** [reply]
大佬，markdown编辑可以支持掘金的选择主题和代码高亮样式么，现在的感觉有点单调呢。

**Reply by Mereithhh** [reply]
以后会支持的…… 暂时可能没人力了

**Comment by Daishu777** [comment]
希望作者能加入对ipfs的支持，类似于 https://matters.news/

**Reply by Mereithhh** [reply]
后面会考虑的，但可能优先级不太高

**Comment by anonsaber** [comment]
希望增加为每篇文章设置自定义链接的功能，而不是现在的 post/1 2 3 4 5 6，让 url 更具可读性，也有利于 seo

**Reply by Mereithhh** [reply]
收到，后面会有的
